src/view/trip-info.js

```diff
@@ -0,0 +1,53 @@
+import { createElement } from '../render.js';
+
+function createHeaderTemplate() {
+  return (`<section class="trip-main__trip-info  trip-info">
+  <div class="trip-info__main">
+    <h1 class="trip-info__title">Amsterdam &mdash; Chamonix &mdash; Geneva</h1>
+
+    <p class="trip-info__dates">Mar 18&nbsp;&mdash;&nbsp;20</p>
+  </div>
+</section>`);
+}
+
+function createTitleTemplate(){
+  return (`<div class="trip-info__main">
+  <h1 class="trip-info__title">Amsterdam &mdash; Chamonix &mdash; Geneva</h1>
+
+  <p class="trip-info__dates">Mar 18&nbsp;&mdash;&nbsp;20</p>
+</div>`);
+}
+function createTatalPrice() {
+  return(`<p class="trip-info__cost">
+  Total: &euro;&nbsp;<span class="trip-info__cost-value">1230</span>
+</p>`);
+}
+
+class newElementHeader {
```

Comment:
Переименовываем и пока не передаем аргументы, шаблон уже тут есть, я понял идею, но лучше не делать настолько универсальный класс, который отвечает за отрисовку разных элементов, к тому же они не инкапсулированы (спрятаны) внутри, а передаются извне
```suggestion
export default class InfoView {
  getTemplate() {
    return createInfoViewTemplate();
  }
  ...
}
```